ts/screens/wallet/ManuallyIdentifyTransactionScreen.tsx

```diff
@@ -0,0 +1,121 @@
+import { none, Option, some } from "fp-ts/lib/Option";
+import {
+  Body,
+  Button,
+  Container,
+  Content,
+  Form,
+  H1,
+  Icon,
+  Input,
+  Item,
+  Label,
+  Left,
+  Text,
+  View
+} from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import AppHeader from "../../components/ui/AppHeader";
+import I18n from "../../i18n";
+import variables from "../../theme/variables";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+type State = Readonly<{
+  TransactionCode: Option<string>;
+  FiscalCode: Option<string>;
+  Amount: Option<string>;
+}>;
+
+const style = StyleSheet.create({
+  content: {
+    paddingRight: variables.contentPadding,
+    paddingLeft: variables.contentPadding
+  }
+});
+
+/**
+ * This screen allows the user inserts manually the data which identify the transaction:
+ * - Numero Avviso, which includes: aux, digit, application code, codice IUV
```

Comment:
@LsKoder @fgiobergia @gunzip pls double check after yesterday's call

Reply:
reference on qrcode data from here:
https://www.agid.gov.it/sites/default/files/repository_files/specifiche_attuative_nodo_2_1_0.pdf